ONSdigital/dp-frontend-renderer
# Ensure taxonomy links match that on main website

### What

- Fix taxonomy issues from the beta

### How to review

- Set environment variable `export TAXONOMY_DOMAIN=https://www.ons.gov.uk`
- Verify that the taxonomy matches that of ons.gov.uk
- Verify that taxonomy successfully redirects on feedback form
- Verify that taxonomy successfully redirects on previous versions list
- Verify that clicking on the change language button redirects you to welsh website homepage

### Who can review

Anyone
